src/Plugin/GraphQL/DataProducer/Entity/EntityLoad.php

```diff
@@ -120,10 +133,10 @@ public function __construct(
    *
    * @return \GraphQL\Deferred
    */
-  public function resolve($type, $id = NULL, $language = NULL, $bundles = NULL, FieldContext $context) {
+  public function resolve($type, $id = NULL, $language = NULL, $bundles = NULL, $access = NULL, AccountInterface $accessUser = NULL, string $operation = NULL, FieldContext $context) {
```

Comment:
$access should be TRUE by default. And $accessOperation should be 'view' by default in the function arguments already.